### PR TITLE
fix: Improve the arrangement of metrics in mobile view in ratelimit overview page

### LINKED ISSUE #2731: Improve the arrangement of metrics in  mobile view  in ratelimit->overview page
### Preliminary Checks

- [X] I have reviewed https://unkey.com/docs for existing features that would solve my problem

- [X] I have searched for existing feature requests: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a feature request directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Is your feature request related to a problem? Please describe.

Vertically arranged the metrics of  metrics in  mobile view  in ratelimit->overview page  

### Describe the solution

image

to this 

image

### Describe alternatives you have considered (if any)

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by BadriVishalPadhy**
## What does this PR do?

Vertically arranged the metrics  in mobile view in ratelimit->overview page

Fixes #2731

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ x] Filled out the "How to test" section in this PR
- [ x] Read [Contributing Guide](./CONTRIBUTING.md)
- [ x] Self-reviewed my own code
- [ x] Commented on my code in hard-to-understand areas
- [ x] Ran `pnpm build`
- [ x] Ran `pnpm fmt`
- [ x] Checked for warnings, there are none
- [ x] Removed all `console.logs`
- [ x] Merged the latest changes from main onto my branch with `git pull origin main`
- [ x] My changes don't cause any responsiveness issues

### Appreciated

- [ x ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

Screenshot 2024-12-10 at 16 22 15

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `TimestampInfo` component for displaying formatted timestamps (local, UTC, and relative).
	- Enhanced the layout and responsiveness of the `RatelimitNamespacePage` component.

- **Bug Fixes**
	- Corrected minor formatting issues in class names for improved responsiveness.

### CODE REVIEW

**Comment by BadriVishalPadhy** [comment]
@ogzhanolguncu check plz

File: apps/dashboard/components/timestamp-info 2.tsx
Comment (ogzhanolguncu) [inline apps/dashboard/components/timestamp-info 2.tsx:1]: Why do we have copy of timestamp-info component here? Feels like it's a filename typo or a mistake.

**Comment by ogzhanolguncu** [comment]
Also, let's add horizontal dividers similar to have we use in vertical design

File: apps/dashboard/components/timestamp-info 2.tsx
Comment (BadriVishalPadhy) [reply apps/dashboard/components/timestamp-info 2.tsx:1]: Yeah will do that

**Comment by BadriVishalPadhy** [comment]
Screenshot 2024-12-11 at 07 24 30

@ogzhanolguncu  Do check . . .

File: apps/dashboard/components/timestamp-info 2.tsx
Comment (ogzhanolguncu) [reply apps/dashboard/components/timestamp-info 2.tsx:1]: @BadriVishalPadhy can you remove the duplicate component?

**Review by ogzhanolguncu — APPROVED** [review]

**Comment by BadriVishalPadhy** [comment]
@ogzhanolguncu hey can i ask u something

**Comment by ogzhanolguncu** [comment]
Sure

**Comment by BadriVishalPadhy** [comment]
@ogzhanolguncu can i work as intern at unkey

**Comment by perkinsjr** [comment]
> @ogzhanolguncu can i work as intern at unkey
> 
> I am actively looking for an internship . . .

I Appreciate your interest in interning at Unkey. We currently don't offer internships as we are a small team and don't have the infrastructure to offer it at this time.

**Comment by BadriVishalPadhy** [comment]
thnks @perkinsjr  if there will be any job roles or intern open please do consider me

**Pull request merged**